cmd/soroban-rpc/internal/test/simulate_transaction_test.go

```diff
@@ -836,3 +838,51 @@ func TestSimulateTransactionBumpAndRestoreFootprint(t *testing.T) {
 	assert.NoError(t, err)
 	sendSuccessfulTransaction(t, client, sourceAccount, tx)
 }
+
+func TestCLI(t *testing.T) {
+	test := NewTest(t)
+	ch := jhttp.NewChannel(test.sorobanRPCURL(), nil)
+	client := jrpc2.NewClient(ch, nil)
+
+	sourceAccount := keypair.Root(StandaloneNetworkPassphrase)
+
+	tx, err := txnbuild.NewTransaction(txnbuild.TransactionParams{
+		SourceAccount: &txnbuild.SimpleAccount{
+			AccountID: keypair.Root(StandaloneNetworkPassphrase).Address(),
+			Sequence:  1,
```

Comment:
I'm pretty sure this should be 0. We got it working before, right?

Reply:
Locally I was getting a seq_num error and this seemed to fix it.

Reply:
ah, ok!